apps/fabric-website/src/redirects.ts

```diff
@@ -11,7 +11,11 @@ export function handleRedirects() {
 function redirectLegacyUrls(hash: string) {
   const redirectMap = {
     '#/customizations/themes': '#/components/customizations/themes',
-    '#/customizations/colors': '#/components/customizations/colors'
+    '#/customizations/colors': '#/components/customizations/colors',
+    '#examples/announced/quickactions': '#/components/announced/quickactions',
+    '#examples/announced/searchresults': '#/components/announced/searchresults',
+    '#examples/announced/lazyloading': '#/components/announced/lazyloading',
+    '#examples/announced/bulkoperations': '#/components/announced/bulkoperations'
```

Comment:
should the Announced local URLs be: `#/examples/...` or does it not matter?

Reply:
It looks like some URLs are `#/examples/...` and some are `#examples/...`.

https://github.com/OfficeDev/office-ui-fabric-react/blob/61d3a3869f804e3e989c51578b07a9c096137cbe/apps/fabric-website-resources/src/AppDefinition.tsx#L30

https://github.com/OfficeDev/office-ui-fabric-react/blob/61d3a3869f804e3e989c51578b07a9c096137cbe/apps/fabric-website-resources/src/AppDefinition.tsx#L211

Not sure what the proper one is here.

Reply:
It looks like there are more (and more recent) cases of `#/examples/...`, so I changed it to that instead.